README.md

```diff
@@ -2,7 +2,7 @@
 
 This repository contains a collection of inspirational, unmaintained [Azure Sphere](https://www.microsoft.com/azure-sphere/) software samples and hardware designs ready to be reused.
 
-Please also see the official Azure Sphere Sample and Azure Sphere Hardware Designs for maintained examples:
+Please also see the official Azure Sphere Samples and Azure Sphere Hardware Designs repos for maintained examples:
```

Comment:
Suggest inverting this sentence so reader knows what it's about first:

For maintained samples, see ...

Reply:
done